packages/quantic/force-app/main/default/lwc/quanticBreadcrumbManager/quanticBreadcrumbManager.js

```diff
@@ -49,12 +49,11 @@ export default class QuanticBreadcrumbManager extends LightningElement {
   }
 
   renderedCallback() {
-    initializeWithHeadless(this, this.engineId, this.initialize.bind(this));
+    initializeWithHeadless(this, this.engineId, this.initialize);
   }
 
   /** @param {import("coveo").SearchEngine} engine */
-  @api
```

Comment:
Regarding the `@api` decorator, it seemed correct to me to have it since the method is being called by another component. Or is it working because Quantic components are deployed as custom code and not as a package?

Reply:
When is the initialize function called by another component? I believe the decorator is a remnant from before we added sibling component support.

Reply:
`initialize` is invoked by `quanticHeadlessLoader`. Well it is not a component per se, but it is still external to the component being initialized.

Reply:
True but @api doesn't have to do with that. From what I understand, the decorator would allow a wrapper component to execute something like
```
const quanticBreadcrumb = this.template.querySelector('c-quantic-breadcrumb');
quanticBreadcrumb.initialize(this.engine);
```
But I don't think we need or want that to happen. Maybe I'm wrong though

Reply:
The [`@api` doc](https://lwc.dev/guide/reference#%40api) says:

> To expose a public method, decorate it with @api. Public methods also define the API for a component.

Since the `initialize` method is called from the outside, I would tend to keep it.

Reply:
Ok then given we aren't agreed on either of the changes introduced here i'll just cancel the PR 😅 
Status quo wins.

Reply:
I've thought about it and I'm still not convinced the @api decorator is appropriate here. It's passed to a util function so it's sort of executed externally but like it says "Public methods also define the API for a component" and imo `initialize` shouldn't be part of the component's API. It would be a prop that, if changed, will likely break the component. Other lwc shouldn't have access to it.

Reply:
Honestly, I was expecting LWC to prevent any code external to the component from accessing a method  unless `@api` is set. If it works in Salesforce with Locker, I guess we're good then.